app/extensions/edm/__init__.py

```diff
@@ -145,8 +146,24 @@ def get_encounters(self, target='default'):
         response = self._get('encounters.list', target=target)
         return response
 
+    def get_encounter_data(self, guid, target='default'):
```

Comment:
OK I'm late to the party here and only just reviewing this but this does feel to be in the wrong place (api/edm as opposed to app/encounters).
There should be a way to make this cleaner.

Reply:
colin and i had a chat about this earlier on -- and wanting to do a bit of moving/cleanup for sure.  we are kind of (or at least my interpretation was!) going to see how the **sighting** (passthru) code plays out, so that can help guide us too -- as that will likely create a lot of duplication/overlap with the passthru stuff encounters need.   so this feedback is definitely noted and we will likely be seeing a bunch of shuffling around here.